docs/source/api/data-management/tables.rst

```diff
@@ -116,6 +116,27 @@ feature_type::ratio
    Represents the type of :capterm:`Ratio feature`.
 
 
+.. _api_tables_sparse_indexing:
+
+Sparse indexing
+---------------
+
+An implementation of the sparse indexing formats.
+
+::
+
+   enum class sparse_indexing { zero_based, one_based };
+
+.. .. namespace:: oneapi::dal
```

Comment:
```suggestion
.. namespace:: oneapi::dal
```

Reply:
@Vika-F I think this and the next typos can be the reason for an error in the build.

Reply:
I think those typos are not the reason of the build failures because there is the same syntax at the lines 80-81 and 103-104.